data/signin.js

```diff
@@ -0,0 +1,265 @@
+class Signin{
+
+    constructor() {
+        this.selector_type = 'dusk';
+
+        this.params = {
+            group:{
+                count: null,
+                name: null,
+            }
+        }
+
+        this.selectors = {
+            email: 'signin-email_or_username',
+            password: 'signin-password',
+            submit: 'signin-signin',
+            page_heading: 'h3',
+            home_page_heading: 'h2',
+            page_subheading: 'p',
+            alert_box: 'alertdialog',
```

Comment:
@satyajeet-s001 is this a selector `alertdialog`?

Reply:
No sir . "alertdialog" is the attribute value which is present in attribute "role" . it is also present in selectors.
refer: https://img-v4.getdemo.dev/screenshot/phpstorm64_CWrQI7GYlv.mp4